src/azure-cli-core/azure/cli/core/_util.py

```diff
@@ -176,6 +177,17 @@ def b64encode(s):
         return encoded.decode('latin-1')
 
 
+def b64_to_hex(s):
```

Comment:
IMO we should avoid adding such util methods to core unless they will really be used by a lot of other modules.
As this PR is, we'd have to release both 'azure-cli-keyvault' and 'azure-cli-core' for this change.